svg/elements/feDiffuseLighting.json

```diff
@@ -0,0 +1,242 @@
+{
+  "svg": {
+    "elements": {
+      "feDiffuseLighting": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/SVG/Element/feDiffuseLighting",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": true
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": true
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": null
+            },
+            "safari": {
+              "version_added": null
+            },
+            "safari_ios": {
+              "version_added": null
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
+        "in": {
```

Comment:
At least `in` and `kernelUnitLength` are required for this element to work, so you should just copy the basic support info to them. This *might* also be the case for `surfaceScale` and `diffuseConstant`, though I'm not that sure about that.

Reply:
Correcting that. `kernelUnitLength` is not required, just `in`. I thought that `kernelUnitLength` sets the coordinates of the light, though that's actually defined by the child elements of `<feDiffuseLighting>`.